details_page.html

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
+<html lang = "en" > 
+   <head> 
+      <meta charset = "utf-8"> 
+      <title>details page</title >
```

Comment:
Default boostrap meta tag is missing
```suggestion
      <meta charset = "utf-8">
      <meta name="viewport" content="width=device-width, initial-scale=1">
      <title>details page</title >
```